board/drivers/can.h

```diff
@@ -235,34 +235,6 @@ void can_set_gmlan(uint8_t bus) {
   }
 }
 
-// TODO: remove
-void can_set_obd(uint8_t harness_orientation, bool obd){
```

Comment:
Where did this go?

Reply:
It was replaced by set_can_mode , isn't it?